mava/wrappers/jumanji.py

```diff
@@ -265,3 +267,104 @@ def observation_spec(self) -> specs.Spec[Union[Observation, ObservationGlobalSta
             return specs.Spec(ObservationGlobalState, "ObservationSpec", **obs_data)
 
         return specs.Spec(Observation, "ObservationSpec", **obs_data)
+
+
+class CleanerWrapper(MultiAgentWrapper):
+    """Multi-agent wrapper for the Cleaner environment."""
+
+    def __init__(self, env: Cleaner, add_global_state: bool = False):
+        super().__init__(env, add_global_state)
+        self._env: Cleaner
+
+    def modify_timestep(self, timestep: TimeStep) -> TimeStep[Observation]:
+        """Modify the timestep for the Cleaner environment."""
+
+        def create_agents_view(grid: chex.Array, agents_locations: chex.Array) -> chex.Array:
+            """Create separate channels for dirty cells, wall cells and agent positions.
+            Also add a channel that marks an agent's own position."""
+
+            num_agents = self.num_agents
+
+            # N: Number of agents
+            # R: Number of grid rows
+            # C: Number of grid columns
+            # grid: (R, C)
+
+            # (N, 2)
+            agents_locations = agents_locations
+
+            # Get dirty / wall tiles from first agent's obs and tile.
+            # (A, R, C)
+            dirty_channel = jnp.tile(jnp.where(grid == DIRTY, 1, 0), (num_agents, 1, 1))
+            wall_channel = jnp.tile(jnp.where(grid == WALL, 1, 0), (num_agents, 1, 1))
+
+            # (2, N)
+            xs, ys = agents_locations[:, 0], agents_locations[:, 1]
+
+            # Mask each agent's position so an agent can idenfity itself.
+            # Sum the masked grids together for global agent information.
+            # (A, R, C)
+            pos_per_agent = jnp.repeat(jnp.zeros_like(grid)[None, :, :], num_agents, axis=0)
```

Comment:
`jnp.newaxis` is a bit clearer than `None`
```suggestion
            pos_per_agent = jnp.repeat(jnp.zeros_like(grid)[jnp.newaxis, :, :], num_agents, axis=0)
```